src/opendr/perception/pose_estimation/hr_pose_estimation/HighResolutionLearner.py

```diff
@@ -0,0 +1,992 @@
+from opendr.engine.constants import OPENDR_SERVER_URL
+
+# OpenDR lightweight_open_pose imports
+
+from opendr.perception.pose_estimation.hr_pose_estimation.filtered_pose import FilteredPose
+from opendr.perception.pose_estimation.hr_pose_estimation.utilities import track_poses
+from opendr.perception.pose_estimation.hr_pose_estimation.algorithm.models.with_mobilenet import \
+    PoseEstimationWithMobileNet
+from opendr.perception.pose_estimation.hr_pose_estimation.algorithm.models.with_mobilenet_v2 import \
+    PoseEstimationWithMobileNetV2
+from opendr.perception.pose_estimation.hr_pose_estimation.algorithm.models.with_shufflenet import \
+    PoseEstimationWithShuffleNet
+
+from opendr.perception.pose_estimation.hr_pose_estimation.algorithm.modules.load_state import \
+    load_state
+from opendr.perception.pose_estimation.hr_pose_estimation.algorithm.modules.keypoints import \
+    extract_keypoints, group_keypoints
+from opendr.perception.pose_estimation.hr_pose_estimation.algorithm.datasets.coco import CocoValDataset
+
+from opendr.perception.pose_estimation.hr_pose_estimation.algorithm.val import \
+    convert_to_coco_format, run_coco_eval, normalize, pad_width
+from opendr.perception.pose_estimation.hr_pose_estimation.algorithm.scripts import \
+    make_val_subset
+
+
+class HighResolutionPoseEstimationLearner(Learner):
+
+    def __init__(self, device='cuda', backbone='mobilenet',
+                 temp_path='temp', mobilenet_use_stride=True, mobilenetv2_width=1.0, shufflenet_groups=3,
+                 num_refinement_stages=2, batches_per_iter=1, base_height=256,
+                 first_pass_height=360, second_pass_height=540,
+                 experiment_name='default', num_workers=8, weights_only=True, output_name='detections.json',
+                 multiscale=False, scales=None, visualize=False,
+                 img_mean=np.array([128, 128, 128], np.float32), img_scale=np.float32(1 / 256), pad_value=(0, 0, 0),
+                 half_precision=False):
+
+        super(HighResolutionPoseEstimationLearner, self).__init__(temp_path=temp_path, device=device, backbone=backbone)
+
+        self.first_pass_height = first_pass_height
+        self.second_pass_height = second_pass_height
+        self.img_resol = 1080  # default value for sample image in OpenDr server
+
+        self.parent_dir = temp_path  # Parent dir should be filled by the user according to README
+
+        self.num_refinement_stages = num_refinement_stages  # How many extra refinement stages to add
+        self.batches_per_iter = batches_per_iter
+
+        self.experiment_name = experiment_name
+        self.num_workers = num_workers
+        self.backbone = backbone.lower()
+        self.half = half_precision
+
+        supportedBackbones = ["mobilenet", "mobilenetv2", "shufflenet"]
+        if self.backbone not in supportedBackbones:
+            raise ValueError(self.backbone + " not a valid backbone. Supported backbones:" + str(supportedBackbones))
```

Comment:
snake_case, also removed mobilenet support
```suggestion
        supported_backbones = ["mobilenetv2", "shufflenet"]
        if self.backbone not in supported_backbones:
            raise ValueError(self.backbone + " not a valid backbone. Supported backbones:" + str(supported_backbones))
```

Reply:
This is wrong, only mobilenet comments should be erased.